### PR TITLE
Ajout de la partie admin / blog, des pages d'erreur, amélioration de …

### PR DESCRIPTION
**Pull request by jdevuono54**

### CODE REVIEW

File: public/scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+$(document).ready(() => {
+    function onKonamiCode(cb) {
+        let input = '';
+        const key = '38384040373937396665';
+
+        document.addEventListener('keydown', e => {
```
Comment (jdevuono54) [inline public/scripts/main.js:6]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
        document.addEventListener('keydown', (e) => {
```

Comment (jdevuono54) [inline public/scripts/main.js:6]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
        document.addEventListener('keydown', (e) => {
```

File: public/scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+      return 1;
+    });
+  }
+
+  onKonamiCode(() => {
+    const img = new Image();
+
+    img.src = `../images/konami/k${Math.floor(Math.random() * 5)}.webp`;
+    img.style.width = '350px';
```
Comment (jdevuono54) [inline public/scripts/main.js:25]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
        img.style.width = "350px";
```

File: public/scripts/main.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    const key = '38384040373937396665';
+
+    document.addEventListener('keydown', (e) => {
+      input += (`${e.keyCode}`);
+
+      if (input === key) {
+        return cb();
+      }
+
+      if (!key.indexOf(input)) return 1;
```
Comment (jdevuono54) [inline public/scripts/main.js:13]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
      if (!key.indexOf(input)) {return 1;}
```

File: public/scripts/main.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+$(document).ready(() => {
+  function onKonamiCode(cb) {
+    let input = '';
+    const key = '38384040373937396665';
+
+    document.addEventListener('keydown', (e) => {
```
Comment (jdevuono54) [inline public/scripts/main.js:6]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
    document.addEventListener("keydown", (e) => {
```

File: public/scripts/main.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  onKonamiCode(() => {
+    const img = new Image();
+
+    img.src = `../images/konami/k${Math.floor(Math.random() * 5)}.webp`;
+    img.style.width = '350px';
+    img.style.height = '300px';
+    img.style.transition = '6s all linear';
+    img.style.position = 'fixed';
```
Comment (jdevuono54) [inline public/scripts/main.js:28]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
        img.style.position = "fixed";
```

File: public/scripts/main.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    const key = "38384040373937396665";
+
+    document.addEventListener("keydown", (e) => {
+      input += (`${e.keyCode}`);
+
+      if (input === key) {
+        return cb();
+      }
+
+      if (!key.indexOf(input)) { return 1 };
```
Comment (jdevuono54) [inline public/scripts/main.js:13]: Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
      if (!key.indexOf(input)) { return 1; };
```

Comment (jdevuono54) [inline public/scripts/main.js:13]: Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
      if (!key.indexOf(input)) { return 1 }
```

File: public/scripts/main.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    const key = "38384040373937396665";
+
+    document.addEventListener("keydown", (e) => {
+      input += (`${e.keyCode}`);
+
+      if (input === key) {
+        return cb();
+      }
+
+      if (!key.indexOf(input)) { return 1 }
```
Comment (jdevuono54) [inline public/scripts/main.js:13]: Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
      if (!key.indexOf(input)) { return 1; }
```

**Review by TheGarious — CHANGES_REQUESTED** [review]

File: composer.json [hunk 8]
```diff
@@ -4,14 +4,21 @@
     "type": "project",
     "license": "MIT",
     "require": {
-		"php": "^7.4",
+        "php": "^7.4",
```
[the remark this answers is not in the document]
Comment (jdevuono54) [reply composer.json]: 174fba360795b482b7e49f6f9b0723fa9532c1fb

**Pull request merged**